index.html

```diff
@@ -27,8 +28,103 @@ <h1 class="main-heading">I'm Misal.<br> Glad to see you!</h1>
           <li><a href="#" class="icons"><img src="images/icons/M.png"></a></li>
           <li><a href="#" class="icons"><img src="images/icons/github.png"></a></li>
           <li><a href="#" class="icons"><img src="images/icons/5.png"></a></li>
-        </ul>
+        </ul><br>
         </div>
       </section>
+    </div>
+    <div id="works-grid">
+      <div id="works-1">
+        <div class="works-contents">
+          <img src="images/Snapshoot.png">
+          <label class="works-heading">Tonic</label>
```

Comment:
- Instead of a `<label>` tag, how about we use a `<hx>` tag? An `<h2>` will be great since this represents the project title. 🤗